cluster/gke/config-common.sh

```diff
@@ -18,7 +18,9 @@
 # Specifically, the following environment variables are assumed:
 # - CLUSTER_NAME  (the name of the cluster)
 
-ZONE="${ZONE:-us-central1-f}"
+if [ -z "${REGION}" ]; then
```

Comment:
What if someone sets both REGION and ZONE? I don't see it validated anywhere.

I think that we either need to validate that only one of those is set (preferred option), or alternatively to  use region if it is set, otherwise use zone (and then leave this as it was).